x/wasm/keeper/msg_server_integration_test.go

```diff
@@ -126,20 +142,8 @@ func TestInstantiateContract(t *testing.T) {
 			}
 
 			// check event
-			events := rsp.Events
-			assert.Equal(t, 2, len(events))
-			assert.Equal(t, "message", events[0].Type)
-			assert.Equal(t, 2, len(events[0].Attributes))
-			assert.Equal(t, "module", string(events[0].Attributes[0].Key))
-			assert.Equal(t, "wasm", string(events[0].Attributes[0].Value))
-			assert.Equal(t, "sender", string(events[0].Attributes[1].Key))
-			assert.Equal(t, myAddress.String(), string(events[0].Attributes[1].Value))
-			assert.Equal(t, "instantiate", events[1].Type)
-			assert.Equal(t, 2, len(events[1].Attributes))
-			assert.Equal(t, "_contract_address", string(events[1].Attributes[0].Key))
-			assert.Contains(t, string(rsp.Data), string(events[1].Attributes[0].Value))
-			assert.Equal(t, "code_id", string(events[1].Attributes[1].Key))
-			assert.Equal(t, "1", string(events[1].Attributes[1].Value))
+			assert.Equal(t, spec.expEvents, rsp.Events)
+			assert.Contains(t, string(rsp.Data), string(rsp.Events[1].Attributes[0].Value))
```

Comment:
This is covered by L92 and should be removed.

Reply:
And you may need check below in L143
```
var instantiateResponse types.MsgInstantiateContractResponse
 require.NoError(t, wasmApp.AppCodec().Unmarshal(rsp.Data, &instantiateResponse))
```

Reply:
Thanks. I fixed it as pointed out.